### PR TITLE
Organize transport

### LINKED ISSUE #30: Change the term `IPC` to `Transport`
`remote-trait-object` is general purpose library, and there is no reason not to allow actual networking (over internet or whatever) as far as it follows the interface `trait Ipc`. In that case, it should be renamed as `trait Transport`.

### PR DESCRIPTION
**Pull request by junha1**
Fixes #30

### CODE REVIEW

File: remote-trait-object-tests/src/test_module.rs [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 mod control_loop;
 
-use cbasesandbox::ipc::intra::Intra;
-use cbasesandbox::ipc::{Ipc, IpcRecv, IpcSend};
+use cbasesandbox::transport::intra::Intra;
+use cbasesandbox::transport::{Ipc, TransportRecv, TransportSend};
```
Comment (majecty) [inline remote-trait-object-tests/src/test_module.rs:20]: Here is `Ipc`

File: remote-trait-object-tests/src/lib.rs [hunk 2]
```diff
@@ -22,6 +22,6 @@ extern crate remote_trait_object_macro as rto_macro;
 mod test_concurrent_ping;
```
Comment (majecty) [inline remote-trait-object-tests/src/lib.rs:22]: In the commit: "Rename ipc to transport"

Could you add a simple description of why "transport" is a better name than "ipc"?

File: remote-trait-object-tests/src/test_module.rs (hunk 1 above)
Comment (junha1) [reply remote-trait-object-tests/src/test_module.rs]: I checked all occurrences of `Ipc`. updated.

File: remote-trait-object-tests/src/lib.rs (hunk 2 above)
Comment (junha1) [reply remote-trait-object-tests/src/lib.rs:22]: done

**Pull request merged**